bases/br_rj_riodejaneiro_brt_gps/registros/table_config.yaml

```
@@ -7,39 +7,42 @@ dataset_id: br_rj_riodejaneiro_brt_gps # AUTO GENERATED
 url_ckan: https://basedosdados.org/dataset/br-rj-riodejaneiro-brt-gps # AUTO GENERATED
 url_github: https://github.com/basedosdados/mais/tree/master/bases/br_rj_riodejaneiro_brt_gps # AUTO GENERATED
 
-version: 0.1 # REQUIRED
+version: <vA.B> # REQUIRED
```

```suggestion
version: 1.0 # REQUIRED
```

em tese essa agora é a versao estavel, entao faz sentido entrar como 1.0